spec/controllers/verify/sessions_controller_spec.rb

```diff
@@ -129,23 +129,25 @@
           post :create, profile: partial_attrs
 
           expect(response).to render_template(:new)
-          expect(flash[:warning]).to_not match(
-            t('idv.modal.sessions.warning_html',
-              accent: "<strong>#{t('idv.modal.sessions.warning_accent')}</strong>")
-          )
+          expect(flash[:warning]).to be_nil
           expect(response.body).to match t('errors.messages.blank')
         end
       end
 
       context 'un-resolvable attributes' do
+        before do
+          allow(Idv::Attempter).to receive(:idv_max_attempts).and_return 3
+        end
+
         let(:bad_attrs) { user_attrs.dup.merge(first_name: 'Bad') }
 
         it 're-renders form' do
           post :create, profile: bad_attrs
 
           expect(flash[:warning]).to match(
             t('idv.modal.sessions.warning_html',
-              accent: "<strong>#{t('idv.modal.sessions.warning_accent')}</strong>")
+              accent: "<strong>#{t('idv.modal.sessions.warning_accent')}</strong>",
+              attempt: t('idv.modal.attempts', count: 2))
```

Comment:
this feels a bit like a magic number because I am not sure *why* it is 2. I think that stubbing `user.idv_attempts` in the `before` would improve that. Even better would be to define a variable `remaining_atttempts = Idv::Attempter.idv_max_attempts - user.idv_attempts` and then use that variable here.